src/mngt/ocf_mngt_cache.c

```diff
@@ -569,6 +576,8 @@ static int ocf_mngt_rebuild_metadata_handle(ocf_parallelize_t parallelize,
 				&context->core[core_id].lines);
 	}
 
+	env_atomic_add(free_lines, &context->free_lines);
```

Comment:
In case of error this value is meaningless anyway. I think caller of `ocf_mngt_rebuild_metadata()` shouldn't expect any meaningful values when completion function is called with non-zero error.